### PR TITLE
[zh] Fix change-runtime-containerd.md title display error

### PR DESCRIPTION
**Pull request by mengjiao-liu**
https://kubernetes.io/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd/ page title display error:
```
title: 将节点上的容器运行时从 Docker Engine 改为 containerd weight: 8 content_type: task
```

### CODE REVIEW

**Comment by tengqm** [comment]
The page is out of sync?

**Comment by mengjiao-liu** [comment]
> The page is out of sync?

I saw it last night. I forgot to sync.😅
I will sync it.

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 1]
```diff
@@ -86,19 +98,19 @@ Instructions for setting up the Docker repository for your respective Linux dist
    ```
 
 <!--
-3. Restart containerd:
```
Comment (zaunist) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:89]: Why we change "3" to "1" ?

Comment (mengjiao-liu) [reply content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:89]: This is the original content of English upstream.

File: content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md [hunk 2]
```diff
@@ -22,27 +27,32 @@ This task outlines the steps needed to update your container runtime to containe
...
 -->
 ## 腾空节点    {#drain-the-node}
 
-```
-# 将 <node-to-drain> 替换为你所要腾空的节点的名称
+```shell
 kubectl drain <node-to-drain> --ignore-daemonsets
 ```
 
+将 <node-to-drain> 替换为你所要腾空的节点的名称
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md:54]: 不加反引号会被当做 HTML Tag，输出为空。

```suggestion
将 `<node-to-drain>` 替换为你所要腾空的节点的名称
```

Comment (mengjiao-liu) [reply content/zh/docs/tasks/administer-cluster/migrating-from-dockershim/change-runtime-containerd.md]: Updated.Thanks.

**Pull request merged**